im_library/src/vocabulary/IM.ts

```diff
@@ -175,4 +156,11 @@ export class IM {
 
   // Config
   public static HAS_CONFIG = IM.NAMESPACE + "hasConfig";
+
+  // SUB VOCABULARIES
+  public static component = componentVocabulary;
```

Comment:
since these are sub vocabularies and static we should have them in capital letters (for consistency as well)

Reply:
Problem with that is there already is an IM.QUERY, IM.EDITOR etc. This was it looked clearer that these were different.